### ISSUE TITLE
QIntersection sometimes does not render elements properly

### ISSUE THREAD

**Issue by hognevevle** [quasarframework/quasar#6463]
**Describe the bug**
Using QIntersection, sometimes when items are re-arranged, items that are moved into the viewport are not being rendered. 

In these cases, I need to scroll down in my list, and then back up for it to be rendered properly. 

**Codepen/jsFiddle/Codesandbox (required)**
https://codepen.io/hognevevle/pen/PoqbZrv

**To Reproduce**
It seems random when the issue appears, so just keep pressing "Change some data" repeatedly. 
Usually pressing the button rather quickly (i.e. not waiting for every animation to complete before next click) seems to help the issue manifest itself quicker.

Eventually you should see the issue, for example as shown below:

![image](https://user-images.githubusercontent.com/2721188/75139851-fbcd6000-56ed-11ea-8359-25b52d961475.png)

For such empty items, you can scroll down in the the list so that the elemement disappears outside the viewport. Then, as you scroll back up, it will render correctly. 

**Expected behavior**
Elements should always render properly.

**Platform (please complete the following information):**
OS: Windows
Node: 12.13.0
NPM: 6.12.0
Yarn: 1.21.1
Browsers: Chrome
iOS: N/A
Android: N/A
Electron: N/A

**Comment by hawkeye64** [comment]
@hognevevle Yeah, that's weird. It's just not rendering the slotted content at all
![image](https://user-images.githubusercontent.com/10262924/75176219-00474800-56f1-11ea-9de3-12f4c99cec34.png)

**Comment by hawkeye64** [comment]
@rstoenescu 
I am working with the UI core code. Just added a console.log to print out the slot. Clicked the button until the issue happened and then this is what I got:
![image](https://user-images.githubusercontent.com/10262924/75192248-17962d80-5711-11ea-89af-c77fe743c47b.png)
In this next scenario, I am printing out what I had before, as well as `this.$slots.default`
![image](https://user-images.githubusercontent.com/10262924/75192440-78256a80-5711-11ea-9558-7a7c76148129.png)

It appears as if the `slot` function in Quasar utils isn't always returning the slot.

**Comment by hawkeye64** [comment]
Got 3 in a row :
![image](https://user-images.githubusercontent.com/10262924/75193059-ba9b7700-5712-11ea-9fdc-fc9b176bf040.png)

**Comment by hawkeye64** [comment]
nope, my bad, it's because `this.showing` ends up false:
![image](https://user-images.githubusercontent.com/10262924/75193225-04845d00-5713-11ea-8113-db1de1faf8e7.png)

**Comment by hawkeye64** [comment]
I am trying to narrow things down. In this image, the one that is not showing up has some peculiar information (it's the second one down). `isIntersecting: true`, `intersectionRation: 1` and `intersectingRect` is valid. What is peculiar, is the `target` which has `div.q-intersection.example-item.conversation-item-move` while all other targets have `div.q-intersection.example-item`. The `conversation-item-move` comes from the css:
```css
.conversation-item-move
  transition: transform .5s
```
and, the `transition-group` uses this:
```html
<transition-group name="conversation-item">
```
And usage is defined here: https://vuejs.org/v2/guide/transitions.html#List-Move-Transitions

So, it appears possibly a bug in the Vue v-move code that never finished with the transition.

![image](https://user-images.githubusercontent.com/10262924/75203099-7916c600-572a-11ea-96cb-5da7b3a9025e.png)

![image](https://user-images.githubusercontent.com/10262924/75202726-6354d100-5729-11ea-8da7-1b09eca669c8.png)

Others that moved and did display properly also have `prevSibling` and `nextSibling`, but the bove does not.

![image](https://user-images.githubusercontent.com/10262924/75203173-ac595500-572a-11ea-8d13-16bbc96adc32.png)

**Comment by hognevevle** [comment]
@hawkeye64 I'm probably getting this wrong, but just for fun I tried to reproduce the transition part in plain Vue, just to see if any transitions were left with the "conversation-item-move" class after the transition should have finished. Doesn't seem to be the case. https://codesandbox.io/s/festive-hooks-rcgh6

**Comment by hawkeye64** [comment]
@hognevevle Seems like @pdanpdan has some ideas

**Comment by rstoenescu** [comment]
Fix will be available in "quasar" v1.9.5
